### PR TITLE
restore DCOS 1.9.0

### PR DESCRIPTION
**Pull request by jackfrancis**
**What this PR does / why we need it**: restore DCOS 1.9.0 for backwards compatibility

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**If applicable**:
- [ ] documentation
- [ ] unit tests
- [ ] tested backward compatibility (ie. deploy with previous version, upgrade with this branch)

**Release note**:

```
restore DCOS 1.9.0
```

### CODE REVIEW

**Comment by jackfrancis** [comment]
@dmitsh I've added some `// TODO` items where it wasn't obvious how to integrate restoration of 1.9.0 w/ new 1.9.8 version

File: pkg/api/convertertoapi.go [hunk 1]
```diff
@@ -527,13 +527,15 @@ func convertV20160930OrchestratorProfile(v20160930 *v20160930.OrchestratorProfil
 	if api.OrchestratorType == Kubernetes {
 		api.OrchestratorVersion = "1.6.9"
 	} else if api.OrchestratorType == DCOS {
+		// TODO restore 1.9.0 here
 		api.OrchestratorVersion = DCOSVersion1Dot9Dot8
```
Comment (dmitsh) [inline pkg/api/convertertoapi.go]: should be DCOSVersion1Dot9Dot0

File: pkg/api/convertertoapi.go [hunk 2]
```diff
@@ -527,13 +527,15 @@ func convertV20160930OrchestratorProfile(v20160930 *v20160930.OrchestratorProfil
...
 	}
 }
 
 func convertV20160330OrchestratorProfile(v20160330 *v20160330.OrchestratorProfile, api *OrchestratorProfile) {
 	api.OrchestratorType = v20160330.OrchestratorType
 	if api.OrchestratorType == DCOS {
+		// TODO restore 1.9.0 here
 		api.OrchestratorVersion = DCOSVersion1Dot9Dot8
```
Comment (dmitsh) [inline pkg/api/convertertoapi.go]: should be DCOSVersion1Dot9Dot0

File: pkg/api/convertertoapi.go [hunk 3]
```diff
@@ -543,6 +545,7 @@ func convertV20170131OrchestratorProfile(v20170131 *v20170131.OrchestratorProfil
 	if api.OrchestratorType == Kubernetes {
 		api.OrchestratorVersion = common.GetDefaultKubernetesVersion()
 	} else if api.OrchestratorType == DCOS {
+		// TODO restore 1.9.0 here
 		api.OrchestratorVersion = DCOSVersion1Dot9Dot8
```
Comment (dmitsh) [inline pkg/api/convertertoapi.go]: should be DCOSVersion1Dot9Dot0

**Comment by jackfrancis** [comment]
thx @dmitsh! done I think it's ready for review

**Pull request merged**